flow/aggregates.py

```diff
@@ -368,11 +360,10 @@ def _validate_and_filter_job(job):
 
 class _DefaultAggregateStore:
     """This class holds the information of the project associated with
-    an operation function aggregated by the default aggregates i.e.
-    :py:class:`aggregator.groupsof(1)`.
+    an operation function using the default aggregator, i.e.
+    ``aggregator.groupsof(1)``.
```

Comment:
This probably won't be parsed correctly by Sphinx. `:py:class:` will create a cross-link to the corresponding class, but we just want a code snippet.